staging/src/k8s.io/code-generator/cmd/client-gen/args/args.go

```diff
@@ -51,6 +62,8 @@ type CustomArgs struct {
 	ClientsetOnly bool
 	// FakeClient determines if client-gen generates the fake clients.
 	FakeClient bool
+	// NamerExceptionsFile points to a YAML file that contains namer exceptions.
+	NamerExceptionsFile string
```

Comment:
I tend to move the whole flag into main.go as `NamerExceptionsFile` is not used at all outside of main.go. So keeping it there will make it structurally simpler. Also for informer-gen.